testgrid/cmd/gatherer/BUILD.bazel

```diff
@@ -0,0 +1,34 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
+
```

Comment:
I'd specifically put this in 'issue_gatherer' or similar for clarity (there's a good chance I or someone else adds a test results gatherer later)